src/bake/DevServer.zig

```diff
@@ -7933,9 +8027,8 @@ const ErrorReportRequest = struct {
         // HMR chunks use this too, but currently do not host their JS code.
         var parsed_source_maps: AutoArrayHashMapUnmanaged(SourceMapStore.Key, ?SourceMapStore.GetResult) = .empty;
         try parsed_source_maps.ensureTotalCapacity(temp_alloc, 4);
-        defer for (parsed_source_maps.values()) |*value| {
-            if (value.*) |*v| v.deinit(ctx.dev);
-        };
+        defer {} // contents stored in arenas
```

Comment:
is this intentional
```suggestion

```

Reply:
Yes. it's a (weird) pattern i started doing where a reviewer might see the init of an arraylist/container and think "ok when's it being deinitialized", and then see there's no defer deinit.

But it's an arena so it's actually fine. so there is no action to be taken to deinit. The comment is the deinitializer.

oh, there isn't actually any other samples of this... i swear i had at least one.